firmware/projects/DemoCan/main.cc

```diff
@@ -0,0 +1,56 @@
+/// @author Samuel Parent
+/// @date 2024-01-16
+
+#include <chrono>
+#include <iostream>
+#include <thread>
+
+#include "bindings.h"
```

Comment:
we don't use `bindings.h` anymore. You should `extern` reference a shared `CanBase` and bind an mcal object to it in `bindings.cc`. See DemoProject on the main branch for an example